blocks/columns/columns.js

```diff
@@ -31,6 +31,45 @@ export function applySplitPercentages(block) {
   }
 }
 
+function applyHorizontalCellAlignment(block) {
```

Comment:
Just wondering if all blocks can benefit from this or is there a reason you'd want to restrict it to columns block?

Reply:
I'm not sure it would apply equally to all blocks. I think it depends a little on the role of the block. For example hero blocks or carousel blocks wouldn't really benefit from it, but other blocks might. I already discussed a little with @dnbute that a columns block with 1 cell could actually already take the role of a text block and / or a centered block 😄 - see https://github.com/hlxsites/sunstar-foundation/issues/9

I would propose that once we need this elsewhere that we can generalise it by putting it in scripts.js. Right now the code depends on existing earlier logic in the columns block that finds out whether a cell is a text cell or not

Reply:
Ok, I definitely see use of this in blocks such as  `text` , `quote`. We can add as need arises.